lib/cfg-parser.c

```diff
@@ -92,7 +92,7 @@ static CfgLexerKeyword main_keywords[] =
   { "stats_max_dynamics", KW_STATS_MAX_DYNAMIC },
   { "min_iw_size_per_reader", KW_MIN_IW_SIZE_PER_READER },
   { "flush_lines",        KW_FLUSH_LINES },
-  { "flush_timeout",      KW_FLUSH_TIMEOUT },
+  { "flush_timeout",      KW_FLUSH_TIMEOUT, KWS_OBSOLETE, "flush_timeout() has been deprecated, remove from your configuration." },
   { "suppress",           KW_SUPPRESS },
```

Comment:
I think we should add this to the description above (as it is displayed to the user).

"Some drivers support batch-timeout() instead that you can specify at the destination level."